paper/short/main_wc995.tex

```diff
@@ -696,13 +707,14 @@ \section{Results}
 
 \section{Conclusion}
 
-Our results show that RC can accurately reconstruct the PTS of complex, non-tonal, tinnitus-like sounds, capturing the most salient features of the target.
+Our results show that RC can accurately reconstruct the frequency spectrum of certain relevant non-tonal tinnitus sounds.
 Reconstruction accuracies observed here are below the simulated ideal, which may be attributed to noisy responses universally observed in applications of RC, and which may be mitigated by further optimizing the experimental protocol, stimulus generation, and reconstruction method.
 For example, recent approaches to improving RC reconstruction methods can boost efficiency, noise robustness and overall accuracy \cite{comptonStimulusWhiteningImproves2022a}.
 Subjects completed the required number of trials within 10 minutes,
 indicating that this procedure can be conducted in a single clinical visit.
 RC may therefore be useful as the basis for a clinical assay to characterize a wider variety of tinnitus percepts than currently possible.
-Future work will focus on validating this approach in patients suffering from tinnitus.
+Future work will focus on validating this approach with larger sample sizes, with more target signals,
+stricter SPL control.
```

Comment:
"Future work will focus on validating this approach with larger sample sizes, more target signals, and stricter sound pressure level control." 

SPL not actually defined anywhere (see comment on line 607).